langchain/src/llms/llama_cpp.ts

```diff
@@ -87,4 +89,29 @@ export class LlamaCpp extends LLM<LlamaCppCallOptions> {
       throw new Error("Error getting prompt completion.");
     }
   }
+
+  async *_streamResponseChunks(
+    prompt: string,
+    // @ts-expect-error - TS6133: 'options' is declared but its value is never read.
+    options: this["ParsedCallOptions"],
+    runManager?: CallbackManagerForLLMRun
+  ): AsyncGenerator<GenerationChunk> {
+    const promptOptions = {
+      temperature: this?.temperature,
```

Comment:
maxTokens should be here too?

Reply:
‘maxTokens’ is set on the prompt method but to stream I’m using the evaluation method directly on the context. Unfortunately ‘maxTokens’ is not an option available on that method :(